src/cluster/src/install.rs

```diff
@@ -871,8 +934,12 @@ impl ClusterInstaller {
                     attempt = i,
                     "Not all SPUs are ready. Waiting",
                 );
-                println!("{} of {} spu ready", ready_spu, spu);
-                sleep(Duration::from_millis(1000)).await;
+                let sleep_ms = 1000 * 2u64.pow(i as u32);
```

Comment:
I don't know if exponential backoff over 30 iterations is a good idea. It doesn't really take a failure case into consideration

Reply:
Rust doesn't fail

Reply:
will reduce iterations